addons/im_livechat/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'im_livechat'---
+---XML records in module 'im_livechat'---
+NEW ir.actions.act_window: im_livechat.im_livechat_canned_response_action
+NEW ir.actions.act_window.view: im_livechat.mail_channel_action_livechat_form
+NEW ir.actions.act_window.view: im_livechat.mail_channel_action_livechat_tree
+DEL ir.actions.client: im_livechat.ir_action_client_open_livechat_menu
```

Comment:
We have already talked about this. This is something that the script is not going to detect. It's simply: noupdate=1 records are not automatically removed if the disappear, and thus, all of these records will remain. Imagine a cron that is logging error each time it's executed because the method that is calling is missing. That's one of the cases of not removing these noupdate=1 records.